projects/packages/autoloader/tests/php/bin/test-coverage.php

```diff
@@ -72,11 +72,8 @@ function count_lines_before_class_keyword( $file ) {
 	}
 
 	// Support both styles of line endings.
-	$newlines = substr_count( $content, "\r\n", 0, $matches[0][0][1] );
-	if ( $newlines > 0 ) {
-		return $newlines + 1;
-	}
-	$newlines = substr_count( $content, "\n", 0, $matches[0][0][1] );
+	$newlines  = substr_count( $content, "\r\n", 0, $matches[0][0][1] );
+	$newlines += substr_count( $content, "\n", 0, $matches[0][0][1] ) - $newlines;
```

Comment:
Could simplify even more, since "\r\n" contains "\n".
```suggestion
	$newlines  = substr_count( $content, "\n", 0, $matches[0][0][1] );
```